kernel-spark/src/main/java/io/delta/kernel/spark/read/SparkMicroBatchStream.java

```diff
@@ -28,26 +29,43 @@
 import java.io.IOException;
 import java.util.*;
 import org.apache.hadoop.conf.Configuration;
+import org.apache.spark.sql.SparkSession;
 import org.apache.spark.sql.connector.read.InputPartition;
 import org.apache.spark.sql.connector.read.PartitionReaderFactory;
 import org.apache.spark.sql.connector.read.streaming.MicroBatchStream;
 import org.apache.spark.sql.connector.read.streaming.Offset;
 import org.apache.spark.sql.delta.DeltaErrors;
+import org.apache.spark.sql.delta.DeltaOptions;
+import org.apache.spark.sql.delta.DeltaStartingVersion;
+import org.apache.spark.sql.delta.StartingVersion;
+import org.apache.spark.sql.delta.StartingVersionLatest$;
+import org.apache.spark.sql.delta.sources.DeltaSQLConf;
 import org.apache.spark.sql.delta.sources.DeltaSourceOffset;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import scala.Option;
 
 public class SparkMicroBatchStream implements MicroBatchStream {
 
+  private static final Logger logger = LoggerFactory.getLogger(SparkMicroBatchStream.class);
+
   private static final Set<DeltaAction> ACTION_SET =
       Collections.unmodifiableSet(
           new HashSet<>(Arrays.asList(DeltaAction.ADD, DeltaAction.REMOVE)));
 
   private final Engine engine;
   private final String tablePath;
+  private final DeltaOptions options;
+  private final StreamingHelper streamingHelper;
+  private final SparkSession spark;
 
-  public SparkMicroBatchStream(String tablePath, Configuration hadoopConf) {
```

Comment:
nit: we can keep the this constructor and set the default spark session as `SparkSession.active` and DeltaOptions as `null`